keel-core/src/main/kotlin/com/netflix/spinnaker/keel/events/ResourceEvent.kt

```diff
@@ -289,6 +291,54 @@ data class ResourceActuationResumed(
   )
 }
 
+/**
+ * Orca task the managed resource has failed.
+ *
+ * @property reason The reason why actuation failed.
+ */
+data class ResourceTaskFailed(
+  override val apiVersion: ApiVersion,
+  override val kind: String,
+  override val id: String,
+  override val application: String,
+  val reason: String?,
+  override val timestamp: Instant
+) : ResourceEvent() {
+  @JsonIgnore
+  override val ignoreRepeatedInHistory = true
```

Comment:
Do we want to ignore repeat events? I think the method that checks this only cares about the type of message, not the contents. So I'd say we probably want to have `ignoreRepeatedInHistory = false`

Reply:
Right, changed it. thanks!